doc/source/dev/gitwash/git_links.inc

```diff
@@ -13,20 +13,17 @@
 .. _github: https://github.com/numpy/numpy
 .. _github help: https://help.github.com
 .. _`install it`: https://git-scm.com/downloads
-.. _subversion: http://subversion.tigris.org/
-.. _git cheat sheet: http://cheat.errtheblog.com/s/git
+.. _subversion: https://subversion.apache.org/
 .. _pro git book: https://git-scm.com/book/
 .. _git svn crash course: https://git.wiki.kernel.org/index.php/GitSvnCrashCourse
 .. _learn.github: https://learn.github.com/
 .. _network graph visualizer: https://github.blog/2008-04-10-say-hello-to-the-network-graph-visualizer/
 .. _git user manual: https://www.kernel.org/pub/software/scm/git/docs/user-manual.html
 .. _git tutorial: https://www.kernel.org/pub/software/scm/git/docs/gittutorial.html
 .. _git community book: https://book.git-scm.com/
-.. _git ready: http://www.gitready.com/
-.. _git casts: http://www.gitcasts.com/
-.. _Fernando's git page: http://www.fperez.org/py4science/git.html
```

Comment:
Both of these links are gone – `gitcasts.com` points to a Japanese site that has nothing to do with Git. There are already lots of links in this page.

Reply:
please remove the defunct ones. I suppose at some point a link-checking CI job, that runs e.g. weekly, might be needed.